velox/dwio/common/tests/utils/E2EFilterTestBase.h

```diff
@@ -47,6 +47,14 @@ class TestingHook : public ValueHook {
     }
   }
 
+  void addValue(vector_size_t row, int128_t value) override {
```

Comment:
Decimal should not be called in hook

Reply:
If I remove this code, the `E2EFilterTest#longDecimalDictionary` and `E2EFilterTest#longDecimalDirect` tests of Parquet will fail. Or should I disable these two tests first and then fix them in another PR?